### DISCUSSION TITLE
End of Support for Basic Authentication in Microsoft SMTP

### DISCUSSION THREAD

**Discussion by StanFlavius** [pretix/pretix#4388] [Ideas]
I face a big problem with SMTP. 
Microsoft has announced that Exchange Online will permanently remove support for Basic Authentication for SMTP (Simple Mail Transfer Protocol) as of September 2025. Basic Authentication will no longer be supported for accessing and sending emails via SMTP and it is recommended transitioning to OAuth 2.0.

Are there any available plugins that support SMTP with OAuth or are there plans to include this feature in future versions?

**Comment by raphaelm** [comment]
We are aware of this, but currently do not have specific plans to invest the time to implement this.

As far as we know, the statements from Microsoft are rather vague as to whether "app passwords" will continue to be usable…

**Comment by vb-03** [comment]
Now this is happening in Google Workspace/GSuite soon - https://workspaceupdates.googleblog.com/2023/09/winding-down-google-sync-and-less-secure-apps-support.html - even though apparently app passwords for SMTP is still supported?

**Reply by pc-coholic** [reply]
> If you have scanners or other devices using simple mail transfer protocol (SMTP) or LSAs to send emails, you’ll need to either: configure them to use OAuth, use an alternative method, **or configure an App Password for use with the device.**

Exactly this - App Password will stay intact.

I can see a future where this will also be disabled at some point in the future, but the outcry of organisations world-wide if their scanner and printer combos won't work anymore is probably just too much right now ;-)

But - and it bears repeating over and over - Google Workspace is not a good solution to use when organizing events and wanting to send mass emails. The limits - even for paid accounts - [are ridiculously low](https://apps.google.com/supportwidget/articlehome?hl=en&article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F166852%3Fhl%3Den&assistant_id=generic-unu&product_context=166852&product_name=UnuFlow&trigger_context=a) unless using the [SMTP-Relay](https://support.google.com/a/answer/2956491#sendinglimitsforrelay&zippy=%2Creview-sending-limits-for-the-smtp-relay-service). And even that can be too low when hosting big events...
